src/main/java/org/eolang/compiler/syntax/Argument.java

```diff
@@ -47,7 +47,7 @@
      * @param arg Argument name
      * @param tpn Type name
      */
-    Argument(final String arg, final String tpn) {
+    public Argument(final String arg, final String tpn) {
```

Comment:
I know we have IDE's and a docstring, but a more sane name for the argument `tpn` and other such would be appreciated